seed/challenges/00-getting-started/getting-started.json

```diff
@@ -697,6 +838,21 @@
           "Finde deine Stadt auf dieser Liste und klick sie an. Du wirst auf die Facebook Gruppe des Campsites deiner Stadt geleitet. Klick den \"Gruppe beitreten\" Button um eine Beitrittsanfrage zu stellen. Jemand aus der Gruppe sollte dich in kürzester Zeit annehmen. Falls deine Stadt nicht auf der Liste ist, scrolle zum Ende des Wiki Artikels für eine Anleitung wie du ein Campsite für deine Stadt erstellst.",
           "https://github.com/FreeCodeCamp/freecodecamp/wiki/List-of-Free-Code-Camp-city-based-Campsites"
         ]
+      ],
+      "TitleIt": "Unisciti ad un gruppo Free Code Camp nella tua città",
+      "DescriptionIt": [
+       [
+         "//i.imgur.com/XugIMb4.jpg",
+         "Una foto di alcuni dei nostri camper riuniti in un bar locale.",
+         "È possibile programmare insieme ad altri campers nella tua città aderendo ad un gruppo freeCodeCamp locale.",
```

Comment:
In my opinion we could avoid the "locale" repetition by replacing the last "locale" with "nelle vicinanze" or "vicino a te".